src/client/Microsoft.Identity.Client/Platforms/Features/WamBroker/WamAdapters.cs

```diff
@@ -26,7 +26,7 @@ internal class WamAdapters
             string authority = overridenAuthority ??
                  authenticationRequestParameters.AuthorityManager.OriginalAuthority.AuthorityInfo.CanonicalAuthority;
             bool validate = authenticationRequestParameters.AuthorityInfo.ValidateAuthority;
-            AddAuthorityParamToRequest(authority, validate, webTokenRequest); ;
+            AddAuthorityParamToRequest(authority, validate, webTokenRequest);
         }
 
         internal static MsalTokenResponse CreateMsalResponseFromWamResponse(
```

Comment:
Needs some tests. See https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/6056ccdb09f62970696af676a9dcfedc63887de6/tests/Microsoft.Identity.Test.Unit/BrokerTests/WamTests.cs#L177 for an example on how to check that a request property is set. Maybe just modify a few existing tests?

Reply:
Done. Updated WAM tests to assert for telemetry headers.